src/legacy/core_plugins/visualizations/public/np_ready/public/legacy/build_pipeline.ts

```diff
@@ -78,11 +82,16 @@ const vislibCharts: string[] = [
   'line',
 ];
 
-export const getSchemas = (vis: Vis, timeRange?: any): Schemas => {
+const getSchemas = (
+  vis: Vis,
+  opts: {
+    timeRange?: any;
+  }
+): Schemas => {
+  const { timeRange } = opts;
   const createSchemaConfig = (accessor: number, agg: IAggConfig): SchemaConfig => {
     if (isDateHistogramBucketAggConfig(agg)) {
       agg.params.timeRange = timeRange;
-      setBounds(agg, true);
```

Comment:
if the failing test is related i would leave this as is, calling `agg.buckets.setBounds()` here and figure this out in a followup pr

Reply:
yeah looks like this was the cause of the data table functional test failure. i've reverted back to setting the time buckets bounds in `getSchemas` -- hopefully that addresses everything.

i added a TODO so we can look into this later.